### DISCUSSION TITLE
OSU-Translations repository

### DISCUSSION THREAD

**Discussion by Blueberryy** [ppy/osu#19056] [Ideas] [answered]
Create it and let community fix ancient machine translation issues

**Comment by nagi-desuuu** [comment, marked as answer]
But there's Crowdin...

**Reply by Blueberryy** [reply]
oh thank you lol
why there's no links to their crowdin project?

**Reply by peppy** [reply]
![49A02042-E55A-40FF-B679-D3D8C3396A89](https://user-images.githubusercontent.com/191335/178096824-2e4f1763-5c78-40f1-a0b2-e92bfb0d269a.jpeg)

**Reply by Blueberryy** [reply]
oh my god pepe himsel f
answered!

**Reply by Blueberryy** [reply]
yeah uh thank you i didn't know that there's  contributing guide
